src/compiler/emitter.ts

```diff
@@ -2777,24 +2805,45 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
                 }
             }
 
+            function emitAliasEqual(name: Identifier): boolean {
+                for (const specifier of exportSpecifiers[name.text]) {
+                    emitStart(specifier.name);
+                    emitContainingModuleName(specifier);
+                    write(".");

```

Comment:
still does not handle default. see https://github.com/Microsoft/TypeScript/blob/master/src/compiler/emitter.ts#L1616-L1624